frontend/src/views/Blacksmith.vue

```diff
@@ -90,75 +92,427 @@
-                  <weapon-icon class="weapon-icon" :weapon="getWeaponToBurn()" />
+                <div class="row">
+                  <div class="col-lg-2"></div>
+                  <div class="col-lg-2 dust-container" align="center">
+                    <div class="dust">
+                      LB: <span class="text-warning">15</span>
+                      <div class="dust-image1"></div>
+                    </div>
+                    <h2 class="text-center">Lesser</h2>
+                    <div class="boxed">
+                      <h2>{{this.getOwnedDust().slice(0, 1).shift()}}</h2>
+                    </div>
+                    <div class="range">
+                      <div class="sliderValue">
+                        <span>100</span>
+                      </div>
+                      <div class="field">
+                        <div class="value left">0</div>
+                        <input id="myRange" type="range" min="0" max="100" value="0" steps="1">
+                        <div class="value right">100</div>
+                      </div>
+                    </div>
+                  </div>
+                  <div class="col-lg-2 dust-container" align="center">
+                    <div class="dust">
+                      4B: <span class="text-warning">30</span>
+                      <div class="dust-image2"></div>
+                    </div>
+                    <h2 class="text-center">Greater</h2>
+                    <div class="boxed">
+                      <h2>{{this.getOwnedDust().slice(1, 2).shift()}}</h2>
+                    </div>
+                    <div class="range">
+                      <div class="sliderValue">
+                        <span>100</span>
+                      </div>
+                      <div class="field">
+                        <div class="value left">0</div>
+                        <input id="myRange2" type="range" min="0" max="25" value="0" steps="1">
+                        <div class="value right">25</div>
+                      </div>
+                      <b-button
+                              variant="primary"
+                              tagname="confirm_forge_weapon"
+                              class="confirmReforge"
+                              @click="useDust()"
+                              v-tooltip="'Update dust used'">
+                        Press to update the amount of dust used.
+                      </b-button>
+                    </div>
+                  </div>
+                  <div class="col-lg-2 weapon-container dust-container" align="center">
+                    <div class="dust">
+                      5B: <span class="text-warning">75</span>
+                      <div class="dust-image3"></div>
+                    </div>
+                    <h2 class="text-center">Powerful</h2>
+                    <div class="boxed">
+                      <h2>{{this.getOwnedDust().slice(2, 3).shift()}}</h2>
```

Comment:
don't do this. do all this logic in the code and display the values in the template